monitoring-custom-metrics/k8s/service-8080.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Could you please rename k8s folder to deployment so it will be same as other examples?

Reply:
done